packages/pageflow/src/editor/views/EditMetaDataView.js

```diff
@@ -30,10 +30,12 @@ export const EditMetaDataView = Marionette.Layout.extend({
     var state = this.options.state || {};
     var features = this.options.features || {};
     var editor = this.options.editor || {};
+    var entryTypeName = (editor.entryType && editor.entryType.name) || 'none';
 
     var configurationEditor = new ConfigurationEditorView({
       model: entry.metadata.configuration,
-      tab: this.options.tab
+      tab: this.options.tab,
+      attributeTranslationKeyPrefixes: ['pageflow.entry_types.' + entryTypeName + '.pageflow/entry_metadata_configuration']
```

Comment:
Key segments of the form `pageflow/...` are used by rails when generating translation keys for namespaced classes like `Pageflow::Page`. Here we are free to choose whatever name we want and are therefore not forced to repeat the word `pageflow` later in the key.

Since this key format is part of Pageflow's public API, I'd try to make it as consistent as possible with other already existing keys that serve similar purposes. Examples from the code base:

1. `'pageflow.editor.files.attributes.' + this.options.fileType.collectionName` - in `EditFileFile.js`.
2. `'pageflow.editor.widgets.attributes.' + this.name` - in `WidgetType.js` used in `EditWidgetView.js`
3. `this.seed.translation_key_prefix + '.page_attributes'` in `PageType.js` used in `EditPageView.js`
4.  `pageflow.storyline_attributes` in `EditStorylineView.js`

Since all existing usages include the word `attributes` in the key, I'd suggest doing the same here.

As discussed, the third variant (`pageflow.before_after.page_attributes`) is problematic since it risks name collisions between page type names and other existing keys of the form `pageflow.*`.

The last two variants also do not make it clear that these are translations used in the editor.

Combining these ideas I'd suggest to either go with:

* `pageflow.editor.entry_metadata_configuration_attributes.<type_name>` - pro: most similar to widgets/files case
* `pageflow.entry_types.<type_name>.editor.entry_metadata_configuration_attributes` - pro: all keys for an entry type live under a common prefix

Whatever you choose, please extend the example in `creating_entry_types.md` to show an example function adding an input and the corresponding translation key that needs to be defined.

Earlier, we talked shortly about adding a spec. This would be nice, but I'd be ok with skipping it here in the interest of getting things merged.

Reply:
I'll go with the second suggestion. As opposed to widgets, entry types are much more likely to later be extracted from the gem. This justifies the different treatment, I'd say